### PR TITLE
Add make-target debsrc again

### PR DESCRIPTION
**Pull request by HRio**
This target is used to build a Debian source package, that can be used
with pbuilder to build .deb files.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -120,6 +121,10 @@ distdeb-build: $(releasedir)/$(releasename).tar.gz
 distdeb-sign:
 	debsign $(releasedir)/$(name)-*_$(version)_all.deb
 
+debsrc: $(releasedir)/$(releasename).tar.gz
```
Comment (dimipaun) [inline Makefile:124]: shouldn't we name this target distdeb-src to fit in with the rest?

**Review by dimipaun — CHANGES_REQUESTED** [review]

**Comment by HRio** [comment]
Not sure, the other targets above uses wget to get a released version ($(releasedir)/$(releasename).tar.gz), this one builds a source package from the local clone.

So its a bit different, also this was the target name for this in the "safekeep svn world" so just reused it ;-)

**Comment by dimipaun** [comment]
I was referring to these targets:

distdeb-build: $(releasedir)/$(releasename).tar.gz
    tar xz -C /tmp -f $<
    cd /tmp/$(releasename) && dpkg-buildpackage -us -uc
    mv /tmp/$(name)-*_$(version)_all.deb $(releasedir)

distdeb-sign:
    debsign $(releasedir)/$(name)-*_$(version)_all.deb

I think a  distdeb-src would fit the pattern nicely, no?

**Comment by dimipaun** [comment]
actually, I now start to see it your way... :)

**Comment by HRio** [comment]
yes pattern wise its fine but they create something suitable for distribution. They are for handling distribution of a tagged and released version. As they use the target
104 $(releasedir)/$(releasename).tar.gz:  
105         wget -q $(gitroot)/$(name)/archive/$(version).tar.gz  
106         tar xz -f $(version).tar.gz
...

but the new target does not package a released version for "official" distribution, instead its more 
of a target for getting a deb-src out of the current working tree.

**Review by dimipaun — APPROVED** [review]

**Pull request merged**